src/fdTransfer.cpp

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright 2021 Yonatan Goldschmidt
```

Comment:
The code to be merged in upstream needs to retain the original copyright headers.

Reply:
You mean
```suggestion
 * Copyright 2021 Andrei Pangin
```
?

Reply:
Yes, please

Reply:
Done